bin/asinit

```diff
@@ -19,12 +37,7 @@ function printHelp() {
   process.exit(0);
 }
 
-const rl = require("readline").createInterface({
-  input: process.stdin,
-  output: process.stdout
-});
-
-const projectDir = path.resolve(process.argv[2]);
+const projectDir = path.resolve(cliOptions.arguments[2]);
```

Comment:
Does this work? I believe the options util strips the first two components (node and script path).

Reply:
I thought so too, but I checked the output of the parse function manually.

Reply:
```ts
const cliOptions = options.parse(process.argv, asinitOptions);
console.log(cliOptions.arguments);
process.exit(0);
```

Outputs:

```
[
  'C:\\Program Files\\nodejs\\node.exe',
  'C:\\Users\\Joshua Tenner\\projects\\test-project\\node_modules\\assemblyscript\\bin\\asinit',
  'thePath'
]
```

Reply:
I see now, see the other comment :)